app/lib/myaccount_version.rb

```diff
@@ -2,17 +2,31 @@
 
 module MyaccountVersion
   def self.version
-    version_from_github || 'not set'
+    retrieve_version
```

Comment:
I am sure I am missing sth here, why not `self.version` just does what `self.retrieve_version`, why another method needed 🤔

Reply:
My thinking was to separate them for code clarity and clear responsibility of methods - also to make the one public method simple and the private methods handle the complexity. I wanted to the `version` method to have one task: get the version. So to do that it calls `retrieve_version`. That method has more responsibility of "resolving the version" lets say. Calling `retrieve_version` `resolve_version` might have been a better name. 

Happy to change this if you want.

Reply:
Is `self.retrieve_version` private? I did not realize that 🤔

My instinct was like to check where else`retrieve_version` is called by but then it is not, idk maybe I am not thinking `retrieve_version` is too complex - totally my perception. Or maybe retrieve method calling `set_version` kinda confuses me, idk. I might be not understanding well, too.

Reply:
Ok, I'll change it. I somehow left out the `private` keyword.  It is meant to go under self.version and above  `VERSION_CACHE_TIME = 300`

Reply:
thanks!